src/interfaces.ts

```diff
@@ -1,14 +1,11 @@
 import { BigNumber, ContractReceipt, Signer } from "ethers";
 
 export interface TableMetadata {
```

Comment:
Lots of clean up of interfaces that are no longer needed and changing of the props to match what the validator sends

Reply:
Perfect. This looks great.